org-roam-capture.el

```diff
@@ -85,7 +85,7 @@ note with the given `ref'.")
 
 (defcustom org-roam-capture-templates
   '(("d" "default" plain (function org-roam-capture--get-point)
-     "%?"
+     "%i%?"
```

Comment:
It seems like the wrong template is modified here. Org-roam-ref uses the `org-roam-capture-ref-templates` variable.